RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -20,10 +20,10 @@ extension DeviceFactorSourceClient: DependencyKey {
 				// FIXME: Uh this aint pretty... but we are short on time.
 				if let overridingSnapshot = maybeSnapshot {
 					let networkID = Radix.Gateway.default.network.id
-					let profile = try Profile(snapshot: overridingSnapshot)
+					let profile = Profile(snapshot: overridingSnapshot)
 					let network = try profile.network(id: networkID)
-					accounts = network.accounts.elements
-					personas = network.personas.elements
+					accounts = network.getAccounts().elements
```

Comment:
`getUnhiddenAccounts()` and `getUnhiddenPersonas()`

Reply:
or `NonHidden`?

Reply:
The default should be to get the visible accounts, a future method to get all also the hidden accounts should have the qualifier